changelog/unreleased/enhancement-admin-settings-users-role-graph-api-usage

```diff
@@ -0,0 +1,6 @@
+Enhancement: Admin settings, users make usage of graph api for role assignments
+
+We've changed the api for the role assignments in the admin settings app to graph api,
```

Comment:
```suggestion
We've switched over to utilizing user role assignments data from the graph api instead of the custom settings api. Especially for the list view this leads to improved performance.
```